python/cuspatial/cuspatial/__init__.py

```diff
@@ -1,3 +1,4 @@
+from cuspatial._version import get_versions
```

Comment:
Can this be `from ._version import get_versions` just to be consistent with the other imports?

Reply:
Yes of course. Done.